prep_local

```diff
@@ -10,7 +10,7 @@ set -o errexit -o nounset -o pipefail
 : ${VIRTUAL_ENV?"Must be run in a python virtual environment"}
 
 # Install the latest version of pip
-pip install pip==8.1.2
+pip install --upgrade pip
```

Comment:
@cmaloney I found that without this change you'll see errors like this in 16.04, Mint 18, etc.

```
+ ./prep_local
Exception:
Traceback (most recent call last):
  File "/tmp/dcos_build_venv/lib/python3.5/site-packages/pip/req/req_install.py", line 1006, in check_if_exists
    self.satisfied_by = pkg_resources.get_distribution(str(no_marker))
  File "/tmp/dcos_build_venv/share/python-wheels/pkg_resources-0.0.0-py2.py3-none-any.whl/pkg_resources/__init__.py", line 535, in get_distribution
    dist = get_provider(dist)
  File "/tmp/dcos_build_venv/share/python-wheels/pkg_resources-0.0.0-py2.py3-none-any.whl/pkg_resources/__init__.py", line 415, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/tmp/dcos_build_venv/share/python-wheels/pkg_resources-0.0.0-py2.py3-none-any.whl/pkg_resources/__init__.py", line 695, in find
    raise VersionConflict(dist, req)
pkg_resources.VersionConflict: (pip 8.1.1 (/tmp/dcos_build_venv/lib/python3.5/site-packages), Requirement.parse('pip==8.1.2'))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/tmp/dcos_build_venv/lib/python3.5/site-packages/pip/basecommand.py", line 209, in main
    status = self.run(options, args)
  File "/tmp/dcos_build_venv/lib/python3.5/site-packages/pip/commands/install.py", line 328, in run
    wb.build(autobuilding=True)
  File "/tmp/dcos_build_venv/lib/python3.5/site-packages/pip/wheel.py", line 748, in build
    self.requirement_set.prepare_files(self.finder)
  File "/tmp/dcos_build_venv/lib/python3.5/site-packages/pip/req/req_set.py", line 360, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/tmp/dcos_build_venv/lib/python3.5/site-packages/pip/req/req_set.py", line 448, in _prepare_file
    req_to_install, finder)
  File "/tmp/dcos_build_venv/lib/python3.5/site-packages/pip/req/req_set.py", line 387, in _check_skip_installed
    req_to_install.check_if_exists()
  File "/tmp/dcos_build_venv/lib/python3.5/site-packages/pip/req/req_install.py", line 1011, in check_if_exists
    self.req.project_name
AttributeError: 'Requirement' object has no attribute 'project_name'
You are using pip version 8.1.1, however version 8.1.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```